### ISSUE TITLE
Not sure how to redo circus activity in lecture

### ISSUE THREAD

**Issue by e0426051** [nus-cs2113-AY2021S1/forum#35]
Hi, I was unable to complete all the steps during the lecture for the circus activity, so I attempted them again. However, I made the mistake of deleting my previous forked repository on Github as I thought it was interfering with my unsuccessful pushes. Currently, the main branch has been updated and hence I was unable to make another branch called "in-lecture". May I know how I should go about solving this issue?

Error:
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

**Comment by daniellimws** [comment]
You can click on "branches" here (which links you to https://github.com/e0426051/circus/branches)

![image](https://user-images.githubusercontent.com/8130120/92322937-274ee100-f067-11ea-8713-7a65b9f0ca9b.png)

Then you should be able to delete the `in-lecture` branch on GitHub

![image](https://user-images.githubusercontent.com/8130120/92322953-3766c080-f067-11ea-9f02-5982c053655a.png)

Then there shouldn't be a problem with pushing anymore.

**Comment by okkhoy** [comment]
@e0426051 pls try @daniellimws 's suggestion and check if things work.

What do you mean by 
> main branch has been updated and hence I was unable to make another branch called "in-lecture".

If you have uncommitted changes you can stash them and create a new branch.

**Comment by e0426051** [comment]
@daniellimws  @okkhoy 
Apologies for the delayed reply. I just realized my issue is the result of my blunder of forking @okkhoy 's Circus fork instead of the one at `https://github.com/nus-cs2113-AY2021S1/circus`. I have since re-done the activity with the correct fork and everything seems to work now.

Appreciate the help provided.

@okkhoy 
I was mistaken that the `https://github.com/nus-cs2113-AY2021S1/circus` project (I probably shouldn't have called that the 'main branch') had been updated with an `in-lecture` branch when in actual fact I accidentally forked your fork instead. Apologies for the inconvenience caused.
